NCS.DSS.GDPRUpdateCustomerInformation/Function/GDPRUtility.cs

```diff
@@ -1,26 +1,26 @@
-using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
-using NCS.DSS.GDPRUpdateCustomerInformation.Services;
+using NCS.DSS.GDPRUtility.Services;
 
-namespace NCS.DSS.GDPRUpdateCustomerInformation.Function
+namespace NCS.DSS.GDPRUtility.Function
 {
-    public class GDPRUpdateCustomerInformation
+    public class GDPRUtility
     {
         private readonly IIdentifyAndAnonymiseDataService _identifyAndAnonymiseDataService;
-        private readonly ILogger<GDPRUpdateCustomerInformation> _logger;
+        private readonly ILogger<GDPRUtility> _logger;
 
-        public GDPRUpdateCustomerInformation(IIdentifyAndAnonymiseDataService identifyAndAnonymiseDataService, ILogger<GDPRUpdateCustomerInformation> logger)
+        public GDPRUtility(IIdentifyAndAnonymiseDataService identifyAndAnonymiseDataService, ILogger<GDPRUtility> logger)
         {
             _identifyAndAnonymiseDataService = identifyAndAnonymiseDataService;
             _logger = logger;
         }
 
-        [Function(nameof(GDPRUpdateCustomerInformation))]
-        public async Task<IActionResult> RunAsync([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req)
+        [Function(nameof(GDPRUtility))]
+        [FixedDelayRetry(3, "00:00:15")]
+        public async Task<IActionResult> RunAsync([TimerTrigger("0 0 2 1 * *")] TimerInfo timer)
```

Comment:
Please add a comment to explain what this trigger means (NCRONTAB)
`{second} {minute} {hour} {day} {month} {day-of-week}`

Reply:
Resolving - thanks Joe